CMakeLists.txt

```diff
@@ -223,6 +223,20 @@ endif()
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${_gRPC_C_CXX_FLAGS}")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${_gRPC_C_CXX_FLAGS}")
 
+#if(NOT CMAKE_BUILD_TYPE)
```

Comment:
please revert this change.

Reply:
This is just for a internal test. Would revert this.

Reply:
The third commit has been reset. Sorry to waste your time, I didn't realize the commit would be sync to this.